docs-site/content/guide/wordpress-integration.md

```diff
@@ -0,0 +1,80 @@
+# WordPress Integration
+
+If you are looking to implement Typesense on a WordPress site, you can use the WordPress plugin [Search With Typesense](https://wordpress.org/plugins/search-with-typesense/). The plugin allows you to index your posts, pages and provides ways to showcase them via Instant Search and Autocomplete features.
+
+This article will walk through the initial installation and configuration of the plugin.
+
+## Minimum Requirements
+- PHP Version 7.4
+- WordPress Version 5.8
+
+There are two methods for installing the plugin.
+1. [Automatic Installation](#automatic-installation)
+2. [Manual Installation](#manual-installation)
+
+## Automatic Installation
+This is the preferred and most simple method to install the plugin.
+- On your WordPress website go to plugins > Add New > Search for "Search with Typesense"
+- Click install and then activate the plugin
+
+## Manual Installation
+If for some reason automatic installation is not possible, go to [https://wordpress.org/plugins/search-with-typesense/](https://wordpress.org/plugins/search-with-typesense/) , and you will see the download button. Clicking download button will provide you with a zip file of the plugin then.
+- Go to WordPress Admin area > Plugins > Add New and click upload plugin.
+- Click upload plugin and then add the zip file
+- The plugin will then be installed, then activate the plugin.
+
+Once the plugin has been installed and activated - we need to do someone initial setup.
+
+## Setup and Configuration
+
+### Setup
```

Comment:
Could you include a screenshot of the WP Admin configuration section for the plugin, with some sample values filled out?

Here's an example of how to include images: https://raw.githubusercontent.com/typesense/typesense-website/master/docs-site/content/guide/dynamodb-full-text-search.md (Search for "images" in that page).

Reply:
Added images / screenshots for overall guide.